### PR TITLE
A new content authoring command: entity:create

### PR DESCRIPTION
**Pull request by weitzman**
`entity:create` introduces a novel interaction pattern (inspired by `git rebase --interactive`). The user populates a yaml document in her text editor. The user may save the document at any time, which triggers entity validation if there are errors, they are reported as comments at the top of the document. The UX feels similar to the iterative cycle of submitting a web form, fixing validation errors, and re-submitting.

Notes:
- Any field with "skip" as its value will ignore validation for that field. 
- YAML comments highlight required fields, and multiple value fields for easier data entry.
- After a successful creation, the URL to the entity is presented.

Watch the video below. Feel free to pause as needed.

[asciicast](https://asciinema.org/a/je9ZXdryLGlRxrizTnX574tcZ). 

```
Create a content entity after prompting for field values.

When entering field values, one may submit an incomplete document and any entity violations
 will be helpfully reported at the top of the document. enter the string 'skip' as
 a value in order to skip validation for that field.

Examples:
 drush entity:create node article --validate=0 Create an article entity and skip validation entirely.
 drush entity:create user user              Create an article omitting a field.
--skip-fields=field_media_image
 drush entity:create user user --editor=nano   Create a user using the Nano text editor.

Arguments:
 entity_type An entity type name.
 bundle      A bundle name

Options:
 --validate[=VALIDATE]     Validate the entity before saving. [default: true]
 --uid=UID                 The entity author ID. Also used by permission checks (e.g. content moderation
 --skip-fields=SKIP-FIELDS A list of field names that skip both data entry and validation. Delimit fields by comma
 --editor=EDITOR           A string of bash which launches user's preferred text editor. Defaults to
                           ${VISUAL-${EDITOR-vi}}.
 --no-validate             Negate --validate option.

Aliases: econ, entity-create
```

### CODE REVIEW

**Comment by weitzman** [comment]
\@todo Integrate entity editing

File: src/Commands/core/EntityCreateCommands.php [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+        return implode("\n", $lines);
+    }
+
+    /**
+     * Show/hide a field when building initial YAML.
+     *
+     * @param FieldDefinitionInterface $instance
+     * @return bool
+     */
+    private function showfield(FieldDefinitionInterface $instance): bool
```
Comment (DieterHolvoet) [inline src/Commands/core/EntityCreateCommands.php]:
```suggestion
    private function showField(FieldDefinitionInterface $instance): bool
```

File: src/Commands/core/EntityCreateCommands.php [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+     * @param ContentEntityInterface $entity
+     * @param array $options
+     */
+    private function getInitialYaml(array $instances, ContentEntityInterface $entity, array $options): string
+    {
+        $lines = $comments = [];
+        // Build field names and default value.
+        foreach ($instances as $field_name => $instance) {
+            $comment = [];
+            if (!$this->showfield($instance)) {
```
Comment (DieterHolvoet) [inline src/Commands/core/EntityCreateCommands.php]:
```suggestion
            if (!$this->showField($instance)) {
```

**Review by DieterHolvoet — COMMENTED** [review]
Looks good!

**Pull request merged**